images/metalk8s-ui/conf/nginx.conf

```diff
@@ -0,0 +1,44 @@
+server {
```

Comment:
This is a duplicate of `ui/conf/nginx.conf`, no?
Do we need to keep these two copies,

Reply:
Hum no, forgot to remove it.